### PR TITLE
add: avatar  and badge component

### PR DESCRIPTION
**Pull request by 05him**

### CODE REVIEW

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  border: 2px solid black;
+  border-radius: 50%;
+  width: 70px;
+  height: 70px;
+}
+
+.avtr-xs {
+  width: 30px;
+  height: 30px;
+}
```
Comment (Anurag021) [inline components/avatar/avatar.css:11]: it would be better if you can add rem or % instead of px for most of the places other than border.

**Review by Anurag021 — APPROVED** [review]
Also it would be better if you can add images on the avatars, rest the code looks good to me.

File: components/avatar/avatar.css (hunk 1 above)
Comment (05him) [reply components/avatar/avatar.css:11]: oky sure will make those changes

**Review by VLeads — APPROVED** [review]
Code looks clean. Go ahead !

**Review by anik31 — APPROVED** [review]
Try to use rems everywhere. Variable naming & code is good. Add screenshots in PR description for the reviewers.

File: components/avatar/avatar.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+.avtr-s {
+  width: 50px;
+  height: 50px;
+}
+
+.avtr-l {
+  width: 90px;
+  height: 90px;
+}
```
Comment (b0llu) [inline components/avatar/avatar.css:21]: you can use rems for values

File: components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  padding: 2px;
+  text-align: center;
+  box-sizing: content-box;
+}
+
+.simple-badge,
+.number-badge {
+  width: 15px;
+  height: 15px;
+  background: red;
```
Comment (b0llu) [inline components/badge/badge.css]: use variables instead of actual color name

**Review by b0llu — APPROVED** [review]
Great work! do try to implement the changes

File: components/avatar/avatar.css (hunk 2 above)
Comment (05him) [reply components/avatar/avatar.css:21]: thanks, did those changes

File: components/badge/badge.css (hunk 3 above)
Comment (05him) [reply components/badge/badge.css]: corrected these things too

**Pull request merged**